packages/forms-web-app/src/views/projects/examination.njk

```diff
@@ -66,6 +66,26 @@
     govuk-section-break--xl
     govuk-section-break--visible"/>
 
+ <h2 class="govuk-heading-m">Contact us for support</h2>
+          <p class="govuk-body"> 
```

Comment:
Looks like this `<p>` is indented a bit much?

Reply:
Hi James, thank you - you're right about formatting, I do have prettier in my docs (I am using VSCode), so I thought it would auto-format once I saved the file?